spring-boot/src/main/java/org/springframework/boot/logging/logback/SpringPropertyAction.java

```diff
@@ -46,13 +48,16 @@ public SpringPropertyAction(Environment environment) {
 	@Override
 	public void begin(InterpretationContext ic, String elementName, Attributes attributes)
 			throws ActionException {
-		String name = attributes.getValue(NAME_ATTRIBUTE);
+		String[] names = attributes.getValue(NAME_ATTRIBUTE).split(COMMA_DELIMITER);
```

Comment:
Can you use `org.springframework.util.StringUtils.commaDelimitedListToStringArray` for this please?